### DISCUSSION TITLE
any way to remove the separator for a single line?

### DISCUSSION THREAD

**Discussion by Blade04208** [fastfetch-cli/fastfetch#1092] [Q&A]
exactly as the title says. ive tried to copy the global `separator:` var at the top and making it blank in a module's vars but it doesnt work.

![image](https://github.com/user-attachments/assets/0f23c552-6e82-448d-bb0b-6b14308c8dc5)

**Comment by CarterLi** [comment]
Try this: `{"type":"title","format":"{#red}| {user-name-colored}{at-symbol-colored}{host-name-colored}"}`
